File: Javascript/Programmers_level/level1/두 개 뽑아서 더하기.js
function solution(numbers) {
    let arr = new Set()
    
    idx = 0
    for (let i = 0; i < numbers.length; i++) {
        for (let j = i + 1; j < numbers.length; j++) {
            arr.add(numbers[i] + numbers[j])
        }
    }
    
    // set을 배열로 바꾸기: Array.from(set)
    const result = Array.from(arr).sort((a, b) => {
        return a - b
    })
    
    return result
    
}

/* 다른 사람의 풀이
// 이렇게도 set을 list로 변환할 수 있음
const answer = [...new Set(temp)]

return answer.sort((a, b) => a - b)
*/

File: python/Programmers_level/level1/두 개 뽑아서 더하기.py
from itertools import permutations 
def solution(numbers):
    result_arr = set()
    # 조합 구하기
    arr = list(permutations(numbers, 2))
    # 결과 set에 저장
    for i in arr:
        result_arr.add(i[0] + i[1] ) 
    return sorted(result_arr)